ci-operator/step-registry/redhat-performance/benchmark-runner/02-operators-deploy/redhat-performance-benchmark-runner-02-operators-deploy-commands.sh

```
@@ -0,0 +1,102 @@
+if [[ -s /secret/expected_nodes ]]; then
+  EXPECTED_NODES=$(<"/secret/expected_nodes")
+  EXPECTED_NODES="${EXPECTED_NODES%$'\n'}"
+fi
+
+# --- CNV Nightly Registration (if Vault secrets exist) ---
+if [[ -s /secret/cnv_nightly_registered ]] && [[ -s /secret/cnv_nightly_catalog_source ]]; then
+  echo "=== Applying CNV nightly catalog source ==="
+  scp ${SSH_ARGS} /secret/cnv_nightly_registered root@"${PROVISION_IP}":/tmp/cnv_registered.sh
+  scp ${SSH_ARGS} /secret/cnv_nightly_catalog_source root@"${PROVISION_IP}":/tmp/catalog_source.yaml
+
+  ssh ${SSH_ARGS} root@"${PROVISION_IP}" bash <<NIGHTLY_EOF
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)